index.html

```diff
@@ -64,6 +64,24 @@ <h1 class="title">Spider Man</h1>
 
     </section>
 
+        <section class="animate__animated animate__backInLeft top-gun">
+        <div class="container">
+            <h1 class="title">Rowing Woman</h1>
+            <p class="description">Sara po burzliwej awanturze z chłopakiem traci dom i środki do życia. Zdesperowana kradnie samochód, który staje się jej jedynym schronieniem. Odkrywając jego zawartość, zakochuje się we właścicielu pojazdu i postanawia go odnaleźć. Wyrusza w podróż przez amerykańskie pustkowia, która pozwoli jej przewartościować życiowe pragnienia i priorytety.</p>
+            <img class="vod-logo" alt="Rowing-woman"
```

Comment:
You can also create images directory in your project and save the images instead of pull the external resource, ok?